src/test/scala/nl/knaw/dans/lib/encode/PathEncodingSpec.scala

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (C) 2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.lib.encode
+
+import java.nio.file.Paths
+
+import org.scalatest.{ FlatSpec, Matchers, OptionValues }
+
+class PathEncodingSpec extends FlatSpec with Matchers with OptionValues {
+
+  "escapePath" should "return a string without any escaped characters when the path segments contain only alphanumeric characters or a '_' character" in {
+    val path = Paths.get("abcdefghijklmnopqrstuvwxyz/ABCDEFGHIJKLMNOPQRSTUVWXYZ/1234567890/_")
```

Comment:
please extract the strings into a single value to really demonstrate that the input is exactly the same as the output.

Reply:
done